object-construction-checker/src/main/java/org/checkerframework/checker/objectconstruction/ObjectConstructionChecker.java

```diff
@@ -53,8 +52,18 @@
     String withoutCMAnno = calledMethodsString.trim().substring("@CalledMethods({".length());
```

Comment:
@kelloggm one confusing thing is that when you have one called method, the annotation looks like `@CalledMethods("setSize")`, but when you have multiple, it looks like `@CalledMethods({"setSize",...})`.  Since we always trim `"@CalledMethods({"` from the beginning I worry that this is working by accident.  Could you tweak the logic or at least add some comments explaining how both cases are handled?

Reply:
I've rewritten the method to hopefully make it much more clear - it wasn't quite working by accident, but it was a mess. Thanks for pushing me to clean it up.